alcs-frontend/src/app/features/notice-of-intent/decision/decision-dialog/decision-dialog.component.html

```diff
@@ -12,7 +12,6 @@ <h2 class="card-title">{{ isEdit ? 'Edit' : 'New' }} Decision</h2>
           [matDatepicker]="datePicker"
           formControlName="date"
           name="date"
-          required
```

Comment:
decision date is not required anymore?

Reply:
It is, but the required/not required should be set on the form control not the HTML when using Reactive Forms.